en/course-04/lesson-01/chapter-02/README.md

```diff
@@ -0,0 +1,71 @@
+# Chapter 2: Understanding Ordinals and the Ordinals Protocol
+
+## Introduction to Ordinals:
+Ordinals are a category of numbers that denote the position or order of an element in a sequence. Examples include "first," "second," "third," and so forth. In the realm of blockchain technology, the term "ordinals" extends beyond its traditional usage to encompass a numbering scheme or protocol associated with organizing and identifying data within the blockchain.
```

Comment:
Maybe elaborate how Ordinals in bitcoin is just a system to assign ID's to satoshis. Then go on how these can be inscribed with some arbitrary chunk of data.